table/two_level_iterator.cc

```diff
@@ -154,14 +154,8 @@ void TwoLevelIterator::Prev() {
 
 
 void TwoLevelIterator::SkipEmptyDataBlocksForward() {
-  while (second_level_iter_.iter() == nullptr ||
-         (!second_level_iter_.Valid() &&
-         !second_level_iter_.status().IsIncomplete())) {
-    // Move to next block
-    if (!first_level_iter_.Valid()) {
```

Comment:
From my debugging experience the `first_level_iter_` was only invalid when `second_level_iter_.iter() == nullptr`. Hence I thought about moving this case out/removing that case.